core/kernel/dt.c

```diff
@@ -316,6 +316,24 @@ int fdt_read_uint32_array(const void *fdt, int node, const char *prop_name,
 	return 0;
 }
 
+int fdt_read_uint32_index(const void *fdt, int node, const char *prop_name,
+			  int index, uint32_t *value)
+{
+	const fdt32_t *cuint = NULL;
+	int len = 0;
+
+	cuint = fdt_getprop(fdt, node, prop_name, &len);
+	if (!cuint)
+		return -FDT_ERR_NOTFOUND;
```

Comment:
`return len;`

Reply:
I added another patch to fix it in fdt_read_uint32_array() as well